tests/unit/intake_rubicon/test_viz.py

```diff
@@ -58,3 +59,38 @@ def test_metric_correlation_plot_source():
     assert visualization.selected_metric == catalog_data_sample["selected_metric"]
 
     source.close()
+
+
+def test_datatable_plot_source():
+    catalog_data_sample = {
+        "dataframe_name": "dataframe_name",
+        "experiments": None,
+        "plotting_func": None,
+        "plotting_func_kwargs": None,
+        "x": None,
+        "y": None,
+    }
+
+    source = DataframePlotDataSource(
+        dataframe_name="dataframe_name",
+        experiments=None,
+        plotting_func=None,
+        plotting_func_kwargs=None,
+        x=None,
+        y=None,
+    )
+    assert source is not None
+
+    source.discover()
+
+    visualization = source.read()
+
+    assert visualization is not None
+    assert visualization.experiments == catalog_data_sample["dataframe_name"]
+    assert visualization.experiments == catalog_data_sample["experiments"]
```

Comment:
```suggestion
    assert visualization.dataframe_name == catalog_data_sample["dataframe_name"]
```

first one needs to be `dataframe_name` and the second needs to be removed. the experiments on the viz object will be actual rubicon experiments and the experiments in the catalog are just ids. they won't match